SQL/New_patches/2018-09-07_EEG_tables.sql

```diff
@@ -18,8 +18,8 @@ CREATE TABLE `physiological_output_type` (
 -- Create a physiological_file table
 CREATE TABLE `physiological_file` (
   `PhysiologicalFileID`       INT(10)    UNSIGNED  NOT NULL      AUTO_INCREMENT,
-  `PhysiologicalModalityID`   TINYINT(3) UNSIGNED  DEFAULT NULL,
-  `Caveat`                    TINYINT(1) UNSIGNED  DEFAULT 0,
+  `PhysiologicalModalityID`   INT(5)     UNSIGNED  DEFAULT NULL,
+  `Caveat`                    INT(5)     UNSIGNED  DEFAULT 0,
```

Comment:
Is this a yes/no field?

Reply:
Actually, this won't be used in the end. Thanks for bringing that up!! I did not see it anymore... Will remove it in the next commit.